### ISSUE TITLE
[Bug]: screen saver time should be configurable

### ISSUE THREAD

**Issue by zcutlip** [ParetoSecurity/pareto-mac#102]
### What happened?

Checking for screen saver time (actually screen lock time!) is good security. However, everyone's threat model is different and 5 minutes may not be appropriate for everyone

The screen saver time should be configurable, at least to a point. Perhaps the check could default to 5 minutes, but allow the user to specify a acceptable time up to 10 minutes, or some other reasonable maximum.

### Version

1.6.29 - 3941 (setapp subscription)

**Comment by zupo** [comment]
Yep, agreed, we just haven't had the time to get to it. I'll bump priority.
